test/ld-preload/libinterceptor.c

```diff
@@ -0,0 +1,25 @@
+#define _GNU_SOURCE
+#include <unistd.h>
+#include <dlfcn.h>
+#include <errno.h>
+
+ssize_t (*original_pread)(int fd, void *buf, size_t count, off_t offset) = NULL;
+
+ssize_t pread64(int fd, void *buf, size_t count, off_t offset)
```

Comment:
I think it is better to have a comment to explain what code path in criu is tested.